Source/Core/Core/HW/SI/SI_DeviceGCController.cpp

```diff
@@ -153,6 +271,32 @@ GCPadStatus CSIDevice_GCController::GetPadStatus()
     pad_status = Pad::GetStatus(m_device_number);
   }
 
+  // Where GCControllerInputPolled callbacks are run when a movie is not being played back.
+  if (Scripting::ScriptUtilities::IsScriptingCoreInitialized() && !Movie::IsPlayingInput())
+  {
+    Scripting::OnGCControllerPolledCallbackAPI::current_game_cube_controller_number_polled =
```

Comment:
That (and all the following members) are a mouthful to type...but I don't really have any better suggestion at the moment.

Feels like this might be better suited for methods (similar to `ScriptUtilities`) or maybe a class that encapsulates all of this (but I'm not really sure about performance and lifetime characteristics in relation to what _else_ you have planned here)